### ISSUE TITLE
CPCA_Block

### ISSUE THREAD

**Issue by NameHedada** [Cuthbert-Huang/CPCANet#4]
Sorry, I didn't find the relevant code section for CACP attention. Can you help me point out the location of the corresponding code file?

**Comment by Cuthbert-Huang** [comment]
Of course. I'm sorry to be a bother to you. It's called RepBlock class in [CPCANet.py](https://github.com/Cuthbert-Huang/CPCANet/blob/main/CPCANet/nnunet/network_architecture/CPCANet.py) (line 87).
